crypto/ts/@types/optimisedmt.d.ts

```diff
@@ -1,25 +1,177 @@
 declare module "optimisedmt" {
-  export class OptimisedMT {
+  export type Leaf = bigint;
+  export type PathElements = bigint[][];
+  export type Indices = number[];
+  interface MerkleProof {
+    pathElements: PathElements;
+    indices: Indices;
+    depth: number;
+    root: bigint;
+    leaf: Leaf;
+  }
+  export class IncrementalTree {
+    leavesPerNode: number;
+
+    depth: number;
+
+    zeroValue: bigint;
+
     root: bigint;
 
     nextIndex: number;
 
+    leaves: Leaf[];
+
+    zeros: bigint[];
+
+    filledSubtrees: bigint[][];
+    /* eslint-disable-next-line */
```

Comment:
You shouldn't use any type and disable it with lint.

Reply:
This is autogenerated when compiling with declarations: true in tsconfig.json - what would you suggest there? manually generate the d.ts file?